core/scripts/cre/environment/docs.md

```diff
@@ -95,12 +95,17 @@ go run main.go env start
 
 # or to start environment with an example workflow
 go run main.go env start --with-example
+
+# or to start environment with an image that contains all supported capabilities
+go run main.go env start --with-plugins-docker-image ***.dkr.ecr.***.amazonaws.com/chainlink:nightly-20250522-plugins
```

Comment:
Can we comment that it's sdlc or stage image, so people know where to search for it? To me AWS account is not a secret, I do understand it's maybe not very secure to put it out here but let's give devs a hint where to look for the image, reference workflow that builds it or at least where ECR is located (sdlc or stage)